gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/jupiter/reactor/handler/context/AbstractExecutionContext.java

```diff
@@ -73,14 +79,47 @@ public Completable interruptWith(ExecutionFailure executionFailure) {
         );
     }
 
+    @Override
+    public Maybe<Buffer> interruptBody() {
+        return interrupt().toMaybe();
+    }
+
+    @Override
+    public Maybe<Buffer> interruptBodyWith(final ExecutionFailure failure) {
+        return interruptWith(failure).toMaybe();
+    }
+
     @Override
     public Flowable<Message> interruptMessages() {
-        return Flowable.error(new InterruptionException());
+        return messagesInterruption().toFlowable();
+    }
+
+    @Override
+    public Maybe<Message> interruptMessage() {
+        return messagesInterruption().toMaybe();
+    }
```

Comment:
same kind of comment as @jhaeyaert 
we have `interruptMessage` (singular) and `messagesInterruption()` (plural)

we also noticed the same "problem" in connector configuration and sometimes it's a little mindblowing or error prone.

maybe we should discuss it altogether to have a better understanding of when to put an "s" or not